semtypes/src/main/java/io/ballerina/types/PredefinedType.java

```diff
@@ -191,6 +191,36 @@ BT_CELL, bddAtom(ATOM_CELL_INNER_MAPPING_RO)
             BT_CELL, bddAtom(ATOM_CELL_INNER_RO)
     );
 
+    private static final CellAtomicType CELL_ATOMIC_OBJECT_MEMBER_KIND = CellAtomicType.from(
+            STRING, CellAtomicType.CellMutability.CELL_MUT_NONE
+    );
+
+    private static final TypeAtom ATOM_CELL_OBJECT_MEMBER_KIND = createTypeAtom(11, CELL_ATOMIC_OBJECT_MEMBER_KIND);
+    private static final CellSemType CELL_SEMTYPE_OBJECT_MEMBER_KIND = (CellSemType) basicSubtype(
+            BT_CELL, bddAtom(ATOM_CELL_OBJECT_MEMBER_KIND)
+    );
+    private static final CellSemType CELL_SEMTYPE_VAL = (CellSemType) basicSubtype(BT_CELL, bddAtom(ATOM_CELL_VAL));
+    private static final CellSemType CELL_SEMTYPE_NEVER = (CellSemType) basicSubtype(BT_CELL, bddAtom(ATOM_CELL_NEVER));
+    private static final MappingAtomicType MAPPING_ATOMIC_OBJECT_MEMBER = MappingAtomicType.from(
+            new String[]{"kind", "value"}, new CellSemType[]{CELL_SEMTYPE_OBJECT_MEMBER_KIND, CELL_SEMTYPE_VAL},
+            CELL_SEMTYPE_NEVER);
+    public static final TypeAtom ATOM_MAPPING_OBJECT_MEMBER = createTypeAtom(9, MAPPING_ATOMIC_OBJECT_MEMBER);
```

Comment:
We need to add the entries for these into `Env.atomTable` similar to,
https://github.com/ballerina-platform/ballerina-lang/blob/00d25157c7d99c6b2d07ccee97db166506d584f8/semtypes/src/main/java/io/ballerina/types/Env.java#L64-L66

It seems that part is missing in the nBallerina codebase too.

Reply:
No longer needed with https://github.com/ballerina-platform/ballerina-lang/pull/43018